tests/cases/standalone/common/copy/copy_from_fs_parquet.sql

```diff
@@ -22,10 +22,18 @@ Copy with_pattern FROM '/tmp/demo/export/parquet/' WITH (PATTERN = 'demo.*');
 
 select * from with_pattern order by ts;
 
+CREATE TABLE with_limit_rows(host string, cpu double, memory double, ts timestamp time index);
+
+Copy with_limit_rows FROM '/tmp/demo/export/parquet/' WITH (MAX_INSERT_ROWS = 10);
```

Comment:
Let's add another case to ensure it works as expected.
```sql
Copy with_limit_rows FROM '/tmp/demo/export/parquet/' WITH (MAX_INSERT_ROWS = 1);
```

Reply:
For small datasets, batch_insert may not accurately limit the number of inserted rows. Perhaps we can create another Parquet file separately so that we can limit the maximum number of inserted rows in the case of small datasets.